lib/src/neu/verifications/contract-mismatcher.ts

```diff
@@ -146,8 +146,11 @@ export class ContractMismatcher {
           return accumulator;
         }
 
-        const result = this.findMismatchOnContent(
-          userInputRequest.headers[userInputHeaderKey],
+        const result = this.findMismatchOnStringContent(
+          {
+            name: userInputHeaderKey,
+            value: userInputRequest.headers[userInputHeaderKey] as string
```

Comment:
is there a way to make this more typesafe?

Reply:
What do you mean by this ? Can you be bit more specific please